desk.css

```diff
@@ -0,0 +1,472 @@
+@font-face {
+  font-family: Poppins;
+  src: url('./fonts/Poppins-Regular.ttf') format('truetype');
+}
+
+@font-face {
+  font-family: Playfair_Display;
+  src: url('./fonts/Playfair_Display/static/PlayfairDisplay-Regular.ttf') format('truetype');
+}
+
+header {
+  display: none;
+}
+
+footer {
+  display: none;
+}
+
+body {
+  margin: 0;
+}
```

Comment:
- [ ] Your mobile version is not properly working. Please work on that. :+1: 

https://user-images.githubusercontent.com/70483835/158980818-da7a5177-a532-4ae2-b4b5-7598e0ab49db.mp4